### PR TITLE
chore: reproduce deployment broadcast files from on-chain data

### PR DESCRIPTION
**Pull request by bowd**
### Description

When I deployed and executed the governance proposal for Mento Governance, I made the immense mistake of not committing the broadcast files to Github. A few days later my macbook went belly up and I lost all the data. Sigh. 

This PR uses an archive node (quicknode) and some manual data to reconstruct the Foundry broadcast files. This is useful to have in conjunction with the rest of our tooling which depends on broadcast files. It would have been very useful a few months ago because it makes it really easy to verify contracts on `celoscan`, I used it to verify a few remaining proxies. 

The script to generate this is a one-off script (famous last words), but I thought it made sense to put it in there in case we ever need something similar, but it shouldn't really be reviewed thoroughly.

### Other changes

N/A

### Tested

Verified broadcast script manually and tried it out with the cleoscan verification script.

### Related issues

N/A

### Backwards compatibility

Precisely 

### Documentation

N/A

### CODE REVIEW

File: bin/reproduce-mainnet-gov.ts [hunk 1]
```diff
@@ -0,0 +1,292 @@
+/*******************************************************************
+ * One-off script used to reproduce the mainned Mento Governance
```
Comment (RyRy79261) [inline bin/reproduce-mainnet-gov.ts:2]:
```suggestion
 * One-off script used to reproduce the mainnet Mento Governance
```

**Review by RyRy79261 — APPROVED** [review]

File: bin/reproduce-mainnet-gov.ts [hunk 2]
```diff
@@ -0,0 +1,292 @@
...
+    contractAddress: trace["result"]["to"],
+    function: null,
+    arguments: null,
+    transaction: {
+      type: tx["typeHex"],
+      from: tx["from"],
+      gas: `0x${tx["gas"].toString(16)}`,
+      value: `0x${tx["value"].toString(16)}`,
+      data: trace["result"]["input"],
+      nonce: tx["nonce"]
```
Comment (baroooo) [inline bin/reproduce-mainnet-gov.ts:257]:
```suggestion
      nonce: `0x${tx["nonce"].toString(16)}`
```

**Review by baroooo — APPROVED** [review]
Added slight modifications to reduce the differences between the foundry generated and script generated broadcast files. Not a blocker for this, feel free to re-gen broadcast files if you see a need.

File: bin/reproduce-mainnet-gov.ts [hunk 3]
```diff
@@ -0,0 +1,292 @@
...
+    const entry = {
+      hash: dt.txHash,
+      transactionType: trace["result"]["type"],
+      contractName: dt.contractName,
+      contractAddress: trace["result"]["to"],
+      function: null,
+      arguments: dt.arguments,
+      transaction: {
+        type: tx["typeHex"],
+        from: tx["from"],
```
Comment (baroooo) [inline bin/reproduce-mainnet-gov.ts:163]:
```suggestion
        from: tx["from"],
        to: tx["to"],
```

**Pull request merged**